Doc/library/fractions.rst

```diff
@@ -87,7 +87,7 @@ another rational number, or from a string.
 
    .. versionchanged:: 3.9
       The :func:`math.gcd` function is now used to normalize the *numerator*
-      and *denominator*. :func:`math.gcd` always return a :class:`int` type.
+      and *denominator*. :func:`math.gcd` always returns an :class:`int` type.
```

Comment:
Not worth changing in this backport, but "an instance of int" would be better than "an int type".